sites/shared/components/workbench/draft/defs.mjs

```diff
@@ -2,28 +2,47 @@ const style = ` style="fill: none; stroke: currentColor;" `
 const grids = {
   imperial: `
       <pattern id="grid" height="25.4" width="25.4" patternUnits="userSpaceOnUse" key="grid">
-        <path class="gridline lg imperial" d="M 0 0 L 0 25.4 L 25.4 25.4" ${style} />
+        <!-- lg: solid lines every 1 inch -->
```

Comment:
I appreciate you want to use comments to make it more clear what's going on, but this is a not an efficient way to do so.
These comments would become part of the SVG defs section, and this would mean that they'd be included in the result of every render call,or each time we generate an SVG or PDF.

Please either include real code comments so they can be optimized out by the bundler, or remove them entirely.